site/content/en/docs/Guides/feature-stages.md

```diff
@@ -23,12 +23,13 @@ A feature within Agones can be in `Alpha`, `Beta` or `Stable` stage.
 that can be found in the [Helm configuration]({{< ref "/docs/Installation/Install Agones/helm.md#configuration" >}}) documentation.
 
 The current set of `alpha` and `beta` feature gates:
+{{% feature publishVersion="1.43.0" %}}
```

Comment:
You'll need to copy&paste the old block and insert this new block, otherwise the <1.43 versions of this page will be missing the table entirely.

Reply:
Almost. You need to both publish it _and_ expire the old version, so the old version gets expired, the new version is published. see https://agones.dev/site/docs/contribute/#within-a-page for the syntax.